From 9ba0b96e4aaa23bb7539c8950ed25850a917d9b3 Mon Sep 17 00:00:00 2001
From: Qiulu Yang <qiulu.xb.yang@sony.com>
Date: Wed, 12 Sep 2018 20:21:48 +0800
Subject: [PATCH 4/4] Increase time limit of search task

When amount of apps is too much, search app info spends much time to
get almost all of app label. So SettingsIntelligence cannot surely get
app info in time (about 300 ms), and gives up the app info result.
Then, user cannot get app info result.

Increase time limit of search task to mitigate it.

Bug: 115603047
Change-Id: Iac1b1e3a3a3b7751b45fb6710e43e84d037b8201
---
 .../settings/intelligence/search/SearchResultAggregator.java    | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/com/android/settings/intelligence/search/SearchResultAggregator.java b/src/com/android/settings/intelligence/search/SearchResultAggregator.java
index 20b156c..2f10c74 100644
--- a/src/com/android/settings/intelligence/search/SearchResultAggregator.java
+++ b/src/com/android/settings/intelligence/search/SearchResultAggregator.java
@@ -30,7 +30,7 @@ public class SearchResultAggregator {
      * Timeout for subsequent tasks to allow for fast returning tasks.
      * TODO(70164062): Tweak the timeout values.
      */
-    private static final long SHORT_CHECK_TASK_TIMEOUT_MS = 300;
+    private static final long SHORT_CHECK_TASK_TIMEOUT_MS = 600;
 
     private static SearchResultAggregator sResultAggregator;
 
-- 
2.17.1

